queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts<3;
```

Comment:
- [ ] This query suppose to list the name of all animals born between the start of 2016 and the end of 2019. Kindly fix that.
[Hint]: it will be like that instead: 
```sql
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-31-12'
```

Reply:
Thank you @Omar-Muhamad for your comments. but for the second one. it will be out of the range. can i get more description please?

Reply:
your query get between the start of 2016 and the start of 2019 and it's supposed to be the end of 2019